File: src/main/kotlin/me/fungames/jfortniteparse/fort/exports/FortItemCategory.java
package me.fungames.jfortniteparse.fort.exports;

import me.fungames.jfortniteparse.fort.objects.ItemCategory;
import me.fungames.jfortniteparse.fort.objects.ItemCategoryMappingData;
import me.fungames.jfortniteparse.ue4.assets.exports.UDataAsset;

import java.util.List;

public class FortItemCategory extends UDataAsset {
    public List<ItemCategoryMappingData> PrimaryCategories;
    public List<ItemCategory> SecondaryCategories;
    public List<ItemCategory> TertiaryCategories;
}

File: src/main/kotlin/me/fungames/jfortniteparse/fort/exports/FortItemCategory.kt
package me.fungames.jfortniteparse.fort.exports

import me.fungames.jfortniteparse.fort.objects.ItemCategory
import me.fungames.jfortniteparse.fort.objects.ItemCategoryMappingData
import me.fungames.jfortniteparse.ue4.assets.exports.UObject
import me.fungames.jfortniteparse.ue4.objects.uobject.FObjectExport

@ExperimentalUnsignedTypes
class FortItemCategory : UObject {
    var PrimaryCategories: Array<ItemCategoryMappingData>? = null
    var SecondaryCategories: Array<ItemCategory>? = null
    var TertiaryCategories: Array<ItemCategory>? = null

    constructor() : super()
    constructor(exportObject: FObjectExport) : super(exportObject)
}